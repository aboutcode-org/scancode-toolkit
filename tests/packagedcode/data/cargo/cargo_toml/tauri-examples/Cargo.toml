[package]
name = "core-api"
version = { workspace = true }
edition = "2021"

[dependencies]
