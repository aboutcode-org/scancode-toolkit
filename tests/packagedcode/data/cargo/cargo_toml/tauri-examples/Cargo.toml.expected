[
  {
    "type": "cargo",
    "namespace": null,
    "name": "core-api",
    "version": null,
    "qualifiers": {},
    "subpath": null,
    "primary_language": "Rust",
    "description": "",
    "release_date": null,
    "parties": [],
    "keywords": [],
    "homepage_url": null,
    "download_url": null,
    "size": null,
    "sha1": null,
    "md5": null,
    "sha256": null,
    "sha512": null,
    "bug_tracking_url": null,
    "code_view_url": null,
    "vcs_url": null,
    "copyright": null,
    "holder": null,
    "declared_license_expression": null,
    "declared_license_expression_spdx": null,
    "license_detections": [],
    "other_license_expression": null,
    "other_license_expression_spdx": null,
    "other_license_detections": [],
    "extracted_license_statement": null,
    "notice_text": null,
    "source_packages": [],
    "file_references": [],
    "is_private": false,
    "is_virtual": false,
    "extra_data": {
      "version": "workspace",
      "rust_edition": "2021"
    },
    "dependencies": [],
    "repository_homepage_url": "https://crates.io/crates/core-api",
    "repository_download_url": null,
    "api_data_url": "https://crates.io/api/v1/crates/core-api",
    "datasource_id": "cargo_toml",
    "purl": "pkg:cargo/core-api"
  }
]